File: yass/src/main/java/entities/Categorie.java
package entities;

import java.sql.Date;

public class Categorie {




    private int idcateg  ;
    private String categevent  ;



    public Categorie() {
    }

    public Categorie(int idcateg, String categevent) {
        this.idcateg = idcateg;
        this.categevent = categevent ;


    }

    public Categorie(String categevent) {
        this.categevent= categevent;

    }




    public int getIdcateg() {
        return idcateg;
    }
    public void setIdcateg(int idcateg) {
        this.idcateg = idcateg;
    }
    public String getCategevent() {
        return categevent ;
    }
    public void setCategevent(String categevent ) {
        this.categevent= categevent ;
    }







    @Override
    public String toString() {
        return "Categprie{" +
                "idcateg=" + idcateg+
                ", categevent=" + categevent + '\'' +
                '}';
    }
}

File: symfony/src/Entity/Categorie.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Categorie
 *
 * @ORM\Table(name="categorie")
 * @ORM\Entity
 */
class Categorie
{
    /**
     * @var int
     *
     * @ORM\Column(name="idcateg", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idcateg;

    /**
     * @var string
     *
     * @ORM\Column(name="categevent", type="string", length=50, nullable=false)
     */
    private $categevent;

    public function getIdcateg(): ?int
    {
        return $this->idcateg;
    }

    public function getCategevent(): ?string
    {
        return $this->categevent;
    }

    public function setCategevent(string $categevent): static
    {
        $this->categevent = $categevent;

        return $this;
    }


}
